.pre-commit-config.yaml

```diff
@@ -64,6 +64,13 @@ repos:
   hooks:
   -   id: check-pre-commit-ci-config
 
+- repo: https://github.com/ansys/pre-commit-hooks
+  rev: v0.3.1
+  hooks:
+    - id: add-license-headers
+      files: '(pyaedt|examples|_unittest|_unittest_ironpython|_unittest_solvers)/.*\.(py)'
+      args:
+      - --start_year=2021
```

Comment:
I imposed this year since it is the one for the first public release of PyAEDT.

Reply:
@jorgepiloto Yes all good.